src/sagemaker/estimator.py

```diff
@@ -350,6 +350,8 @@ def deploy(self, initial_instance_count, instance_type, accelerator_type=None, e
             update_endpoint (bool): Flag to update the model in an existing Amazon SageMaker endpoint.
                 If True, this will deploy a new EndpointConfig to an already existing endpoint and delete resources
                 corresponding to the previous EndpointConfig. Default: False
+            tags(List[dict[str, str]]): Optional. The list of tags to attach to this specific endpoint.
```

Comment:
Do keys in the dict have to be 'Key' and 'Value'? I think we should give an example here.